### ISSUE TITLE
Email_spec causes rspec to fail (though not under Spring)

### ISSUE THREAD

**Issue by davidham** [email-spec/email-spec#187]
I installed email_spec per the instructions, but when I run rspec it fails with the following message:

```
/usr/local/bundle/gems/actionmailer-5.0.0.1/lib/action_mailer/rescuable.rb:6:in `<module:Rescuable>': uninitialized constant ActiveSupport::Rescuable (NameError)
    from /usr/local/bundle/gems/actionmailer-5.0.0.1/lib/action_mailer/rescuable.rb:4:in `<module:ActionMailer>'
    from /usr/local/bundle/gems/actionmailer-5.0.0.1/lib/action_mailer/rescuable.rb:1:in `<top (required)>'
    from /usr/local/bundle/gems/actionmailer-5.0.0.1/lib/action_mailer/base.rb:8:in `require'
    from /usr/local/bundle/gems/actionmailer-5.0.0.1/lib/action_mailer/base.rb:8:in `<top (required)>'
    from /usr/local/bundle/gems/email_spec-2.1.0/lib/email_spec/deliveries.rb:76:in `require'
    from /usr/local/bundle/gems/email_spec-2.1.0/lib/email_spec/deliveries.rb:76:in `<module:Deliveries>'
    from /usr/local/bundle/gems/email_spec-2.1.0/lib/email_spec/deliveries.rb:72:in `<module:EmailSpec>'
    from /usr/local/bundle/gems/email_spec-2.1.0/lib/email_spec/deliveries.rb:1:in `<top (required)>'
    from /usr/local/bundle/gems/email_spec-2.1.0/lib/email_spec.rb:10:in `require'
    from /usr/local/bundle/gems/email_spec-2.1.0/lib/email_spec.rb:10:in `<top (required)>'
    from /app/spec/spec_helper.rb:4:in `require'
    from /app/spec/spec_helper.rb:4:in `<top (required)>'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/configuration.rb:1394:in `require'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/configuration.rb:1394:in `block in requires='
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/configuration.rb:1394:in `each'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/configuration.rb:1394:in `requires='
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/configuration_options.rb:112:in `block in process_options_into'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/configuration_options.rb:111:in `each'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/configuration_options.rb:111:in `process_options_into'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/configuration_options.rb:21:in `configure'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/runner.rb:99:in `setup'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/runner.rb:86:in `run'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/runner.rb:71:in `run'
    from /usr/local/bundle/gems/rspec-core-3.5.2/lib/rspec/core/runner.rb:45:in `invoke'
    from /usr/local/bundle/gems/rspec-core-3.5.2/exe/rspec:4:in `<top (required)>'
    from /usr/local/bundle/bin/rspec:17:in `load'
    from /usr/local/bundle/bin/rspec:17:in `<main>'
```

This is from my `spec_helper.rb`:

```
require 'action_mailer'
require 'email_spec'
require 'email_spec/rspec'
```

When I comment out the email_spec lines, Rspec runs again.

Strange thing though: when I run Rspec under Spring with `bin/rspec`, I don't see this issue. Any ideas why?

**Comment by michaelrshannon** [comment]
@davidham I'm also seeing this under Rails 5 - did you have any luck solving this?

**Comment by davidham** [comment]
@michaelrshannon I'm afraid not, after trying many different things I had to uninstall it. I'm having a hard time with my email-related integration tests generally. I have a feature spec that is supposed to verify (among other things) that an email was sent, but the spec fails intermittently.

**Comment by sergiopantoja** [comment]
@davidham @michaelrshannon Here's the fix. I was running into the same problem and finally traced it down.

In your `rails_helper.rb`, you need to require `spec_helper` **after** loading the environment. So change this:
```ruby
require 'spec_helper'
require File.expand_path('../../config/environment', __FILE__)
```
to this:
```ruby
require File.expand_path('../../config/environment', __FILE__)
require 'spec_helper'
```

It's safe to do this, and in fact this is how RSpec generates the `rails_helper.rb` file by default now anyways (see https://github.com/rspec/rspec-rails/commit/b913f6acfc2c4b7eb53335c4e396c13eed25a96c for reference).

**Comment by davidham** [comment]
@sergiopantoja This is awesome, thanks for posting this! I wanted to use email-spec and I still have it commented out in my Gemfile, but I'll definitely give this a try.

**Comment by caseyprovost** [comment]
:+1 I am still getting this error with the proposed fix above in Rails 5.0.2
